src/routes/safe/store/models/confirmation.js

```diff
@@ -8,12 +8,14 @@ export type ConfirmationProps = {
   owner: Owner,
   type: TxServiceType,
   hash: string,
+  signature: string,
```

Comment:
Should this be 'string?' ? Since it is nullable

Reply:
It's initialised with an empty string so it should be a string type in that case. 
You could also do `signature?: string` and leave out the default value initialisation. Both of these approaches are fine imo

Reply:
I will change this, since it should be `null` by default